### PR TITLE
Fix circular frame overflow issue

### PR DESCRIPTION
**Pull request by smitraj-uniqode**
### Summary

Fix circular frame overflow issue

### Description

- The SVG's viewBox attribute was using outdated canvas dimensions that didn't account for the stroke width of the circular border.
- Fixed the viewBox calculation to use actual canvas dimensions instead of hardcoded values.

### Related Links / Trello / Intercom / Github

https://trello.com/c/GtlzqDeL

### Affected Service(s) / Component(s)

Generator

### Type of change (use labels) - bug / refactor / feature …

Bug

### How have I tested this?

Tested it on staging environment for both POLO and Uniqode

### Risks

- [x] **low** : backward compatible,new feature or patches which do not break existing functionality.

- [ ] **medium** : existing functionality in our domain got altered but this does not break contracts with other domains.

- [ ] **high** : breaking contracts with other domains.

### PR Dependency
Link of PR(s) that need to be merged before this PR.
None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "mobstac-awesome-qr",
-    "version": "2.6.1",
+    "version": "2.6.2-beta.3",
```
Comment (himanshu-mobstac) [inline package.json:3]: @smitraj-uniqode Reminder to update version.

File: src/Svg.ts [hunk 2]
```diff
@@ -272,22 +272,14 @@ export class SVGDrawing {
...
-                        // Using canvasWidth since canvasWidth and canvasHeight are same 
-                        let viewBoxHeight = canvasWidth;
-                        if( this.config.showBarcode) {
-                            viewBoxHeight = viewBoxHeight + 400;
-                        }   
-                        if( this.config.showBarcodeValue  && this.isFrameCircularOrNone) {
-                            viewBoxHeight = viewBoxHeight + 200;
-                        }
-                        //@ts-ignore
-                        canvas.viewbox(0, 0, canvasWidth + 190, viewBoxHeight + 190)
```
Comment (akshay-uniqode) [inline src/Svg.ts:289]: What was the purpose of the 190 offset for this line?

**Comment by himanshu-mobstac** [comment]
## Overall Review

This is a solid fix for the circular frame overflow issue! The approach of using dynamic canvas dimensions instead of hardcoded offsets is much more robust and maintainable.

### ✅ **Strengths:**
- **Dynamic viewBox calculation**: Using `canvas.width()` and `canvas.height()` instead of hardcoded values
- **Consistent padding**: Introducing a calculated padding value throughout the circular frame logic
- **Improved circle positioning**: Better circle creation with explicit diameter and positioning
- **Comprehensive fix**: Addresses the issue across multiple related functions

### 🤔 **Questions & Suggestions:**
1. **Historical context**: What was the original purpose of the `190` offset that was removed? Understanding this helps ensure we're not missing edge cases.

2. **Testing coverage**: Given this affects visual rendering, consider adding:
   - Unit tests for the new padding calculations
   - Visual regression tests for circular frames with different stroke widths
   - Tests with `showBarcode` and `showBarcodeValue` enabled

3. **Documentation**: The padding calculation (`moduleSize / 2`) could benefit from inline comments explaining the rationale.

4. **Consistency check**: Verify that all size parameters passed to helper functions are consistent with the new canvas dimensions.

### 📋 **Testing Checklist:**
- [ ] Circular frames render without overflow
- [ ] Stroke width variations work correctly  
- [ ] Background images in circular frames position properly
- [ ] Barcode and barcode value display correctly with circular frames
- [ ] No regression in other frame types

Overall, this looks like a well-thought-out fix. The main suggestions are around documentation and ensuring comprehensive test coverage.

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (himanshu-mobstac) [inline package.json:1]: **Version bump looks good!** 

Using a beta version (`2.6.2-beta.4`) is appropriate for this fix since it changes visual rendering behavior. This allows for testing before the final release.

**Question**: Is this the 4th beta iteration for version 2.6.2? If so, what were the previous beta changes? This context would help understand the full scope of changes in this version.

File: src/Svg.ts [hunk 4]
```diff
@@ -272,22 +272,14 @@ export class SVGDrawing {
...
+                        canvas.viewbox(0, 0, canvas.width(), canvas.height())
+                    }
                     // @ts-ignore
                     canvas.width(null);
                     // @ts-ignore
                     canvas.height(null);
-                    if(frameStyle === QRCodeFrame.CIRCULAR){
-                        // Using canvasWidth since canvasWidth and canvasHeight are same 
-                        let viewBoxHeight = canvasWidth;
-                        if( this.config.showBarcode) {
```
[the remark this answers is not in the document]
Comment (smitraj-uniqode) [reply src/Svg.ts:282]: Added
